### DISCUSSION TITLE
Security of autoupdate?

### DISCUSSION THREAD

**Discussion by pck1980** [zed-industries/zed#9636] [Feature Requests]
I was wondering if Zed performs some sort of integrity/signature checks of the updates it automatically downloads. From a quick look at the code, it seems to me as if doesn't, but neither did I look in depth, nor do I know much about Rust ;-) 
Does anyone know more about this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
